Create LICENSE

dannysauer opened the pull request:
Bring license into sync with upstream.
Refs fpytloun/docker-pulp#1

fpytloun committed:
Create LICENSE

Closes: #1

fpytloun commented:
@thmo BTW: I briefly checked your modifications against upstream repository and I would be open to merge PR with your changes if you submit it.

dannysauer committed:
Use exec in entrypoint scripts

fpytloun committed:
Merge pull request #2 from Kong/feat/fix_entrypoints_for_upstream

Use exec in entrypoint scripts

fpytloun committed:
Update to pulp 3.20.0, fedora:36 base

This is huge version leap and not backward compatible!

- resource manager worker is no more
- Redis is optional, PULP_CACHE_ENABLED must be set
- db encryption key is needed, set PULP_DB_ENCRYPTION_KEY_VALUE

fpytloun committed:
Add tag push into release make target

fpytloun committed:
Fix pulp uid/gid, update tini

fpytloun committed:
Update to latest pulp and fedora:38
